README.md

```diff
@@ -9,7 +9,7 @@ Creating tool for migrations in magento, with `rollback` option.
 ### Download
 You can add these module into your project by adding lines below into your composer.json `require` section:
 ```
-        "pcf/magento-migration" : "dev-master",
```

Comment:
Imo module is still under development

Reply:
but working version is already downloadable.

Reply:
downloadable didnt mean it is stable and functional :)